R/dash.R

```diff
@@ -287,11 +287,17 @@ Dash <- R6::R6Class(
         output_value <- getStackTrace(do.call(callback, callback_args),
                                       debug = private$debug,
                                       pruned_errors = private$pruned_errors)
-  
+        
```

Comment:
oops trailing whitespace...

Reply:
🙈

Reply:
fixed in 01bc431b8f11711a6566f3d5c9b40054eade11b0